### PR TITLE
Add bitcoin network slides

### PR DESCRIPTION
**Pull request by matnad**
# Description

Add initial version of the bitcoin network slides. 

# ToDo:
- [x] SPV
- [x] Recommended Literature

## Maybe
- [x] Details for Protocol
- [x] Incentive Problems
- [x] More on Historic Number of Nodes

## Type of Change

Please delete options that are not relevant.
- [x] Major feature (New files or assets, large changes to files)

# Checklist:

- [x] I have performed a self-review of my own changes
- [x] I have made sure that any changed LaTeX files compile properly
- [x] I have assigned at least one reviewer

### CODE REVIEW

File: slides/bitcoin_network.tex [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+\begin{frame}{Joining the Network}
+	\begin{center}
+		\begin{tikzpicture}[scale=1, every node/.style={scale=1}]
+			\input{../assets/figures/join_network}
+		\end{tikzpicture}
+	\end{center}
+
+	\vspace{1cm}
+	
+	After the initial handshake, we request a list of addresses and connect to more peers.
```
Comment (Chainomics) [inline slides/bitcoin_network.tex:47]:
```suggestion
	After the initial handshake, we request a list of IP addresses and connect to additional peers.
```

File: assets/figures/node_functions.tex [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	\node at (0, 1.2) {Optional};
+	
+	\node at (1) {\includegraphics[height = 0.1\textheight]{../assets/images/copy}};
+	\node[below = 14pt] at (1) {Copy of Ledger};
+	
+	\node at (2) {\includegraphics[height = 0.1\textheight]{../assets/images/verify}};
+	\node[below = 14pt] at (2) {Verify};
+	
+	\node at (3) {\includegraphics[height = 0.1\textheight]{../assets/images/forward}};
+	\node[below = 14pt] at (3) {Distribute};
```
Comment (Chainomics) [inline assets/figures/node_functions.tex:26]:
```suggestion
	\node[below = 14pt] at (3) {Relay};
```

File: slides/bitcoin_network.tex [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+		\input{../assets/figures/full_node_functions2}
+	\end{tikzpicture}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Wallet Functionality}
+	\centering
+	\begin{tikzpicture}[scale=1, every node/.style={scale=1}]
+		\input{../assets/figures/wallet_functions}
```
Comment (Chainomics) [inline slides/bitcoin_network.tex:83]: Encrypted Key Store. Also please remove the term security layer.

File: slides/bitcoin_network.tex [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+	\end{tikzpicture}
+\end{frame}
+%%%
+
+\begin{frame}{Mining Functionality}
+	\centering
+	\begin{tikzpicture}[scale=1, every node/.style={scale=1}]
+		\input{../assets/figures/mining_functions}
```
Comment (Chainomics) [inline slides/bitcoin_network.tex:91]: Create block -> Assemble Blocks
Also, please change order, i.e. assemble blocks in the left and compute on the right.

File: slides/bitcoin_network.tex [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+%%%
+
+%%%
+\begin{frame}{Network Partitions}
+	\centering
+	\begin{tikzpicture}[scale=1, every node/.style={scale=1}]
+		\input{../assets/figures/partitioned_network}
+	\end{tikzpicture}
+\end{frame}
+%%%
```
Comment (Chainomics) [inline slides/bitcoin_network.tex:136]:
```suggestion
```

File: slides/bitcoin_network.tex [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+
+%%%
+\begin{frame}{Bitcoin is a Peer-to-Peer Network}
+	\centering
+	\begin{tikzpicture}[scale=1, every node/.style={scale=1}]
+		\input{../assets/figures/p2p}
+		\color{focus}
+		\node (text) at (-0.5,3.5) { Nodes };
+		\draw[->, thick]	(text.south) -- (agenta.west);
+		\draw[->, thick]	(text.south) -- (agentb.north west);
```
Comment (Chainomics) [inline slides/bitcoin_network.tex:32]:
```suggestion
```

File: slides/bitcoin_network.tex [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+	\centering
+	\begin{tikzpicture}[scale=1, every node/.style={scale=1}]
+		\input{../assets/figures/extended_network}
+	\end{tikzpicture}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Simplified Payment Verification}
+
```
Comment (Chainomics) [inline slides/bitcoin_network.tex:150]: Less Centralized options:
* Simplified Payment Verification (SPV).
* Running full node at home and connect to it with mobile devices.

File: slides/bitcoin_network.tex [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Pool Mining}
```
Comment (Chainomics) [inline slides/bitcoin_network.tex:155]: Move up.

File: slides/bitcoin_network.tex [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+		Source: \link \url{https://blockchain.info/pools}, March 2021
+	\end{figure}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Recommended Literature}
+	
+\end{frame}
+%%%
```
Comment (Chainomics) [inline slides/bitcoin_network.tex:169]:
```suggestion
```

**Review by Chainomics — CHANGES_REQUESTED** [review]
As discussed.

**Comment by matnad** [comment]
Requested changes are implemented, ready for review.

**Review by Chainomics — APPROVED** [review]
Thank you for the changes @matnad 
Looks good!

**Pull request merged**